### DISCUSSION TITLE
Unexpected drop shadow darkening upon selection

### DISCUSSION THREAD

**Discussion by drinkspiller** [fabricjs/fabric.js#10107] [General]
I have a Group nested within another Group. The main group is added to the Fabric canvas. In order for the inner group to fire 'scaling' events, the outer group needs to set `{subTargetCheck: true, interactive: true}`

Strangely, the addition of `{subTargetCheck: true, interactive: true}` to the outer group has the unexpected side effect of the inner group's drop shadow becoming darker when its parent group is selected. You can see what I mean in this demo:

https://codepen.io/drinkspiller/pen/eYwjbjp?editors=1011

When the card is selected, the drop shadow gets darker. If `{subTargetCheck: true, interactive: true}` is commented out, the drop shadow no longer gets darker when the card is selected, but the 'resizing' handler stops firing. Is there a configuration change that can be made to correct this darker drop shadow upon selection or is this perhaps a bug? Thanks!

**Comment by asturur** [comment]
The darkening of the shadow seems 100% a bug i didn't expect that.

For what regards the event firing, i want to ask you is it clear to you why it fires in that case?

When the group is interactive you are selecting card 1 inside main.
When the group is not interactive you are selecting main.

Is not that the events are not firing, but the fact is that you are dragging a different set of controls in the 2 cases

**Reply by drinkspiller** [reply]
Thanks for the reply, @asturur, and for the explanation as well. Would you like me to convert this discussion to an issue? Also, do you have any suggestions for a workaround that might mitigate the darkening behavior? 

- Mr. Furley

**Reply by asturur** [reply]
i converted to an issue already with the automated button, as of now i have no idea why that is happening, otherwise i would have done a quick hotfix.
